com.unity.cinemachine/Documentation~/CinemachineUsingSplinePaths.md

```diff
@@ -1,4 +1,6 @@
-# Using Spline paths
+# Constrain a camera to a path
+
+Use a Spline to constrain the movement of a camera along a predefined and customizable path.
 
 A Spline path is a path formed by a Spline in a scene. Use a Spline to specify a fixed course to position or animate a CinemachineCamera. Use the [Spline Dolly](CinemachineSplineDolly.md) behaviour to make your CinemachineCamera follow a Spline path.
```

Comment:
We should add here: By default, Unity Splines contain no rotation data.  Camera rotation will be inferred from the spline tangent at any point, combined with the world Up vector.  To add rotation about the tangent, you can use the CinemachineSplineRoll behaviour.  This allows you to specify Roll values for any point along the spline.  Roll values are used to rotate the camera about the spline tangent at that point, giving you more control over the camera's rotation.  

If a CinemachineSplineRoll behaviour is added to the spline, all cameras that use the spline will see it.  Alternatively, you can add the CinemachineSplineRoll behaviour to the CinemachineCamera itself, in which case the roll will only apply to it.

Reply:
Different file, but same comment as above-above. _Carte blanche_ for you.